modules/FormModel.js

```diff
@@ -322,40 +495,111 @@
     checkIfModelsAreStale: function() {
       var staleModels = {},
         currentHashValues = this.__generateAllHashValues();
-      _.each(this.__getAllModels(true), function(model) {
+      _.each(this.getTrackedModels(), function(model) {
         this.isModelStale(model, staleModels, currentHashValues);
       }, this);
       return _.values(staleModels);
     },
 
+    /************** Private methods **************/
+
     /**
-     * Sets up a listener to update the form model if the model's field changes.
+     * Sets up a listener to update the form model if the model's field (or any field) changes.
      * @param model {Backbone.Model} the object model from which this form model will start listen to changes
-     * @param field {String} the field name that it will start listening to.
-     * @method listenToModelField
+     * @param [field] {String} the field name that it will start listening to. If no field is given, it will listen to the general 'change' event.
+     * @method __listenToModelField
+     * @private
      */
-    listenToModelField: function(model, field) {
-      var eventName = 'change:' + field;
-      this.listenTo(model, eventName, _.bind(this.__updateFormField,
-          {formModel: this, field: field}));
+    __listenToModelField: function(model, field) {
+      var eventName = field ? ('change:' + field) : 'change';
+      var callback = field ? _.bind(this.__updateFormField, {formModel: this, field: field}) : this.__updateFormModel;
+      this.listenTo(model, eventName, callback);
       this.__currentUpdateEvents.push({model: model, eventName: eventName});
     },
 
     /**
-     * Sets up a listener on one of the fields that is needed to update a computed value
-     * @param computedConfig {Object} the Computed Config that will be updated when changes occur
+     * Sets up a listener on one (or all) of the fields that is needed to update a computed value
      * @param model {Backbone.Model} the object model from which this form model will start listen to changes
-     * @param field {String} the field name that it will start listening to.
-     * @method listenToComputedValuesDependency
+     * @param [field] {String} the field name that it will start listening to. If no field is given, it will listen to the general 'change' event.
+     * @param computedAlias {String} the name/alias of the computed mapping being used.
+     * @method __listenToComputedValuesDependency
+     * @private
      */
-    listenToComputedValuesDependency: function(computedConfig, model, field) {
-      var eventName = 'change:' + field;
-      this.listenTo(model, 'change:' + field, _.bind(this.__invokeComputedPull,
-          {formModel: this, models: computedConfig.models, pull: computedConfig.pull}));
+    __listenToComputedValuesDependency: function(model, field, computedAlias) {
+      var eventName = field ? ('change:' + field) : 'change';
+      this.listenTo(model, eventName, _.bind(this.__invokeComputedPull,
```

Comment:
because backbone already provides the first (and second i think) argument to event listener callbacks on model changes